src/app/learn/reference/cli-commands/certs/index.html

```diff
@@ -0,0 +1,12 @@
+<stache
+  pageTitle="certs">
+
+  <p>
+    <sky-code>skyux certs</sky-code> manages the local certificate necessary to securely run <sky-code>skyux serve</sky-code> and <sky-code>skyux build --serve</sky-code>.
```

Comment:
Based on the announcement you wrote up, doesn't this manage the certificate for `skyux e2e` also?

Reply:
Actually, just noticed in the SSL Certificate docs that I think this should at least have an entry for a `trust` flag. Are there others?

Reply:
Yes it should also include `skyux e2e`.  Sorry for forgetting that.  The valid "actions" for `skyux certs` are `trust`, `untrust`, and `validate`.  https://github.com/blackbaud/skyux-sdk-cli/pull/36/files#diff-d88d4e80b80304a80915b1354c73cddbR72-R84